test-case/manual/DPMST.md

```diff
@@ -0,0 +1,67 @@
+# DPMST test
+DisplayPort Multi-Stream Transport is a manual test.
+
+# Preconditions
+1. Hardware requirements met for each DPMST case
+
+# Test Description
+* Test output from one test unit to two separate monitors with audio output via
+  DP (as well as with USB-C and HDMI adapters if monitors have those inputs
+  available).
+
+# 1) Single DP -> Multi DP hub Test Case:
+## Hardware Requirements
+1. One DisplayPort hub = single DP -> multi-DP
+2. Two monitors with DP input and audio output
+3. Two DP-USB-C adapters (if monitors also have USB-C input)
+4. Two DP-HDMI adapters (if monitors also have HDMI input)
+
+## Run Instructions
+1. Two monitors are connected via one DP hub using DP-DP cables.
+2. Run:
+   ```
+   amixer contents | grep -i jack -A 2
+   ```
+   to ensure DP jacks on DP hub are detected.
+3. Run aplay, using the 2 DP jacks found with `amixer contents` command.
+   ```
+   aplay -D $dev(jack1) -r $rate -c 2 -f $fmt -d 3 resource/raw/test<specified>.raw
+   aplay -D $dev(jack2) -r $rate -c 2 -f $fmt -d 3 resource/raw/test<specified>.raw
+   ```
+4. Connected monitors should both show display and play audio.
+5. Check dmesg for any unexpected errors.
+* If monitors have USB-C / HDMI input as well, test can also be run with:
+  * DP hub -> DP cables + DP-USB-C adapters -> monitors
+  * DP hub -> DP cables + DP-HDMI adapters -> monitors
+
+# 2) Daisy Chain Test Case:
+## Hardware Requirements
+1. One monitor with DP input **AND** ouput, as well as audio output (monitor1)
+2. An additional monitor with DP input and audio output (monitor2)
+3. Two DP-USB-C adapters (if monitors also have USB-C input)
+4. Two DP-HDMI adapters (if monitors also have HDMI input)
+
+## Run Instructions
+1. Monitors are connected via:
+   test unit -> DP cable -> monitor1 -> DP cable -> monitor2
+2. Run:
+   ```
+   amixer contents | grep -i jack -A 2
+   ```
```

Comment:
We also need amixer settings here.

Reply:
@keqiaozhang after switch to HDA HDMI machine driver, do we still need amixer setting

Reply:
Confirmed that there're no Pin x/Port x kcontrols in amixer setting. so we don't need amixer setting anymore. This feature is provided by Kai in kernel side.

Reply:
awesome --- Thanks for checking into that!